src/tokens-dist/json/css-variables-nested.json

```diff
@@ -185,24 +185,24 @@
         "background": {
           "neutral": {
             "default": {
-              "@": "var(--eds-theme-color-background-neutral-default)",
-              "hover": "var(--eds-theme-color-background-neutral-default-hover)"
+              "hover": "var(--eds-theme-color-background-neutral-default-hover)",
+              "default": "var(--eds-theme-color-background-neutral-default)"
```

Comment:
Can you show how this would be used with Tailwind utility classes?

Reply:
let me check..thanks

Reply:
@jinlee93  adding Tailwind utility classes is working after replacing "@" to "default" in css-variables-nested.json ,please check and let me know thanks